src/test/java/com/google/devtools/build/bfg/BUILD

```diff
@@ -1,18 +1,6 @@
 licenses(["notice"])  # Apache 2.0
 
 java_test(
-    name = "DotFileParserTest",
```

Comment:
is it customary to test protobuffs?

Reply:
do you mean if's customary to test the proto parser?
the parser has tons of tests in its own codebase, so no. 
That's one big reason to use other's well tested code, you don't need to test it yourself :)

Reply:
> do you mean if's customary to test the proto parser?

Somewhat. What I was more concerned about was how we were unable to detect that [`dotfile.isEmpty()`](https://github.com/bazelbuild/BUILD_file_generator/pull/13/commits/f096a9f74c3715e5580fa214e9f56a556952b6d6#diff-d7c317053500b3e4d1c069b1440392a9L111) was not removed. Does bfg currently have e2e tests? 

> That's one big reason to use other's well tested code, you don't need to test it yourself :)
SGTM

Reply:
Agreed, there are no e2e tests which would've caught it.
My approach is to hold off e2e tests as long as possible because they're really annoying to set up, at least inside Google. I think it's still fine to hold off, because there's little complexity in Bfg.java.

Reply:
> My approach is to hold off e2e tests as long as possible because they're really annoying to set up, at least inside Google. I think it's still fine to hold off, because there's little complexity in Bfg.java.

SGTM